closeio_api/__init__.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, base_url, api_key=None, tz_offset=None, max_retries=5,
 
         self.session.headers.update({
             'Content-Type': 'application/json',
-            'X-TZ-Offset': self.tz_offset
+            'X-TZ-Offset': str(self.tz_offset)
```

Comment:
I see we're already wrapping `local_tz_offset()` in `str` above. So instead of `self.tz_offset = tz_offset or str(local_tz_offset())` do `self.tz_offset = str(tz_offset or local_tz_offset())